docs/architecture/adrs/002-streaming-response-aggregation.md

```diff
@@ -0,0 +1,43 @@
+# ADR 002: Streaming Response Behavior and Result Aggregation
+
+This ADR defines the orchestrator strategy of result aggregation and behavior observed by end users of the initial streaming API endpoint (i.e. streaming text generation with classification) when detections are made on streamed output of text generation models.
+
```

Comment:
Can we make a note that these are only considering the `/contents` detectors and not other ATM